src/_includes/layouts/form-vacature.liquid

```diff
@@ -49,11 +49,11 @@
                     </p>
                     <p class="form-element">
                     <label for="form-1-input-5" class="form-label">Factuuradres</label>
-                    <textarea name="Factuuradres" id="form-1-input-5" cols="50" rows="5" class="small" required></textarea>
+                    <textarea name="Factuuradres" id="form-1-input-5" cols="50" rows="5" class="small" autocomplete="street-address" required></textarea>
```

Comment:
Consider `billing street-address` to give a hint for user agents.

Also:

> A street address. This can be multiple lines of text, and should fully identify the location of the address within its second administrative level (typically a city or town), but should not include the city name, ZIP or postal code, or country name.

so you'll need to collect other data separately.